src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -1210,4 +1231,27 @@ private static void CopyDirectory(string sourceDir, string targetDir)
 				CopyDirectory(directory, Path.Combine(targetDir, Path.GetFileName(directory)));
 		}
 	}
+
+	internal class BookButtonInfo
+	{
+		private readonly BookInfo _bookInfo;
+		internal BookInfo BookInfo { get { return _bookInfo; } }
+
+		private readonly BookCollection _collection;
+
+		// isVernacular meaning part of the current TheOneEditableCollection
+		private readonly bool _isVernacularBook;
```

Comment:
Ha! Good catch. It's even easier than that! _bookInfo already has an IsEditable property (with the same definition). So now BookButtonInfo.IsEditable returns _bookInfo.IsEditable.